### PR TITLE
Update reconstructive banner implementation

### PR DESCRIPTION
**Pull request by ibnesayeed**
It implements #449 and #144, but the `Link` parsing is not done yet so navigational buttons will not work. However, the banner is ready to function if the values are supplied.

### CODE REVIEW

**Comment by ibnesayeed** [comment]
Link parsing is implemented as well.

**Comment by machawk1** [comment]
@ibnesayeed The datetime is no longer displayed. Is this intentional?

462c1ddf4e7a288cde40470148c3a9f6d95a69eb

screen shot 2018-07-30 at 11 21 24 am

@ibnesayeed The datetime is not displayed

**Comment by machawk1** [comment]
@ibnesayeed Disregard my above comment. My browser was using a stale SW/reconstruct.

**Comment by machawk1** [comment]
@ibnesayeed Thoughts on changing the logo in the banner to ipwb?

**Comment by machawk1** [comment]
Great work on this, btw!

File: ipwb/reconstructive-banner.js [hunk 1]
```diff
@@ -54,19 +54,25 @@ class ReconstructiveBanner extends HTMLElement {
       if (diff < 0) {
         return 'Capture from the future!';
       }
+      let unit, quotient;
       if (diff >= 31622400000) {
-        return `Captured ${Math.round(diff / 31622400000)} year(s) ago`;
+        unit = 'year';
```
Comment (machawk1) [inline ipwb/reconstructive-banner.js]: We should give semantics to these magic numbers in the future.

Comment (ibnesayeed) [reply ipwb/reconstructive-banner.js]: These numbers are milliseconds in respected bigger units of time that are not going to change. For readability purpose we can define them as constants if you prefer that way.

**Comment by ibnesayeed** [comment]
> Thoughts on changing the logo in the banner to ipwb?

I made it configurable, because it was obvious that we will eventually use the branding of the archive not the banner element. So, go ahead and try it by changing the config in the `serviceWorker.js` file. Logo dimensions might be an issue, but that can be fixed later.

File: ipwb/reconstructive-banner.js (hunk 1 above)
Comment (machawk1) [reply ipwb/reconstructive-banner.js]: @ibnesayeed Yes, defining them as constants would be an improvement.

Comment (ibnesayeed) [reply ipwb/reconstructive-banner.js]: I have made this change in the upstream repo. Will synchronize here later.

**Pull request merged**